### DISCUSSION TITLE
How to clear all xmake packages installed locally?

### DISCUSSION THREAD

**Discussion by yh-sb** [xmake-io/xmake#5378] [Q&A] [answered]
I found that xmake is storing packages in the following paths:
* C:\Users\Y\AppData\Local\.xmake\cache\packages
* C:\Users\Y\AppData\Local\.xmake\packages

Some of them are outdated for me and never be used in future.
**How to entirely clear all of them?**

**Comment by waruqi** [comment, marked as answer]
```console
xrepo clean
```
